examples/multi-many.rs

```diff
@@ -0,0 +1,36 @@
+use std::thread;
```

Comment:
We already have a lot of examples. Would it be feasible to change one of the existing examples instead of adding yet another one? All of these amount to relatively low-value extra code that needs to be maintained.

Reply:
done